index/matchtree.go

```diff
@@ -1435,3 +1457,18 @@ func isRegexpAll(r *syntax.Regexp) bool {
 		return false
 	}
 }
+
+func queryMetaCacheKey(field string, value *regexp.Regexp) string {
+	sum := sha256.Sum256([]byte(fmt.Sprintf("%s:%s", field, value.String())))
```

Comment:
how many unique values do you have? Is the point of the sha is you have a lot and the regexp can be large? TBH I would consider just making the key something like `struct{ field, value string }`.

Reply:
> how many unique values do you have?

We have roughly 8-10  unique  `repoIDs` and `meta` conditions right now.

> TBH I would consider just making the key something like struct{ field, value string }.

Thanks for the call out, I applied this suggestion. 👍